src/components/InventoryDetail/__snapshots__/AppInfo.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AppInfo should render correctly 1`] = `ReactWrapper {}`;
```

Comment:
I am afraid the snapshot is not correct: it's just an empty object. Did you mount the component the right way?